en-us.yml

```
@@ -33,4 +77,42 @@ commands:
     - "Maybe not."
     - "Yes!"
     - "No!"
-    - "¯\\_(ツ)_/¯"
\ No newline at end of file
+    - "¯\\_(ツ)_/¯"
+    vemdezap:
+    description: "Good morning catilangaiada! Come from Whatsapp baby, use the power of Whatsapp in your message to show how 🔝 with the Whatsapp''s girls!"
+    examples:
+    - "come from Whatsapp baby"
+    - "Hi catilangaiada, nobody says nothing kkkkkkkkkkkk"

```

```suggestion
    - "Hi catilangaiada, nobody says nothing hahahaha"
```
```suggestion
    - "Hi catilangaiada, nobody says nothing lmao"
```

não existe esse tipo de risada pra quem n fala portugues, só n sei qual é o melhor